### PR TITLE
[bugfix] cast to int when calling setQuality

### PR DESCRIPTION
**Pull request by Daniel-Ateles**
[bugfix] Stronger typing of int in Config::setQuality clashes when config is read from request where everything is an array of string. Therefore this added cast to int before calling setQuality.

  
  

## Changes in this pull request  
Resolves #

### CODE REVIEW

**Comment by fashxp** [comment]
thx very much ... can you please rebase to 11.1. thx

**Comment by fashxp** [comment]
please rebase to 11.x, thx very much.

File: models/Document/Editable/Area.php [hunk 1]
```diff
@@ -240,7 +240,7 @@ public function isEmpty(): bool
      *
      *
      */
-    public function getElement(string $name): Model\Document\Editable
+    public function getElement(string $name): ?Model\Document\Editable
```
Comment (brusch) [inline models/Document/Editable/Area.php:243]: This is unfortunately a BC-break.

File: bundles/SimpleBackendSearchBundle/src/Model/Search/Backend/Data.php [hunk 2]
```diff
@@ -252,7 +252,7 @@ public function getUserOwner(): ?int
     /**
      * @return $this
      */
-    public function setUserOwner(int $userOwner): static
+    public function setUserOwner(?int $userOwner): static
```
Comment (brusch) [inline bundles/SimpleBackendSearchBundle/src/Model/Search/Backend/Data.php:255]: This seems to be unrelated?

**Review by brusch — CHANGES_REQUESTED** [review]

File: models/Document/Editable/Area.php (hunk 1 above)
Comment (jdreesen) [reply models/Document/Editable/Area.php:243]: Is it really? :thinking:
I don't think widening the *return* type in parent classes causes any problems because covariance allows narrowing the *return* types in child classes. See: https://3v4l.org/UqClY

Comment (brusch) [reply models/Document/Editable/Area.php:243]: Ah, yeah cool, wasn't entirely sure, but makes sense - thanks! 👍
